plugins/FeedsUserProcessor.inc

```diff
@@ -520,18 +506,12 @@ class FeedsUserProcessor extends FeedsProcessor {
     unset($allowed_roles_options[BACKDROP_AUTHENTICATED_ROLE]);
 
```

Comment:
It may be helpful to provide the machine name in the widget UI for reference (see image).

This can be done at this point as follows:
```suggestion
    array_walk($allowed_roles_options, function(&$label, $name) {$label = $label .  " ($name)";});
```

<img width="296" alt="Screen Shot 2022-07-12 at 12 11 47 PM" src="https://user-images.githubusercontent.com/9938978/178565862-46053c81-bfcb-4b60-bb39-a0facf3f2aef.png">

Reply:
Thanks - I considered showing them too but was too tired to figure it out now.